lib/slime/renderer.ex

```diff
@@ -25,10 +40,13 @@ defmodule Slime.Renderer do
   Note that this method of rendering is substantially slower than rendering
   precompiled templates created with Slime.function_from_file/5 and
   Slime.function_from_string/5.
+
+  Note: A HEEx-aware version of render/4 was not included because it would require importing
+  Phoenix.LiveView.HTMLEngine, creating a dependency on Phoenix.
   """
   def render(slime, bindings \\ [], opts \\ []) do
     slime
-    |> precompile
+    |> precompile()
```

Comment:
```suggestion
    |> precompile
```

Removing a formatting change to simplify the diff.